index.html

```diff
@@ -3318,110 +3367,136 @@ <h4>Semantic Interoperability</h4>
+When including a link to an external resource in a [=verifiable credential=], it
+is desirable to know whether the resource has been modified after the
+[=verifiable credential=] was issued. This applies to cases where there is an
+external resource that is remotely retrieved, as well as to cases where the
+[=issuer=] and/or [=verifier=] might have local cached copies of a resource. It
+is also desirable to know that the contents of the JSON-LD context(s) used in
+the [=verifiable credential=] are the same when used by both the [=issuer=] and
+[=verifier=].
         </p>
+
         <p class="issue" title="Mandatory listing of contexts in relatedResouce are under debate.">
 The requirement that contexts be listed in `relatedResource` is currently being
 debated in the VCWG. This requirement might be removed in future iterations of
 the specification.
         </p>
+
         <p>
-Each object in the `relatedResource` array MUST contain the
-following: the [[URL]] to the resource named `id` and the
-<code id="defn-digestSRI">digestSRI</code> information for the resource
-constructed using the method specified in
-<a href="https://www.w3.org/TR/SRI/#integrity-metadata">Subresource Integrity</a>.
+To extend integrity protection to a related resource, an [=issuer=] of a
+[=verifiable credential=] MAY include the `relatedResource` property:
         </p>
+
+        <dl>
+          <dt id="defn-relatedResource">relatedResource</dt>
+          <dd>
+The value of the `relatedResource` property MUST be associated with one or
+more objects of the following form:
+            <table class="simple">
+              <thead>
+                <th>Property</th>
+                <th>Description</th>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>`id`</td>
+                  <td>
+The identifier for the resource is REQUIRED and conforms to the format defined
+in Section [[[#identifiers]]]. The value MUST be unique among the list of
+related resource objects.
+                  </td>
+                </tr>
+                <tr>
+                  <td>`mediaType`</td>
+                  <td>
+An OPTIONAL valid media type as listed in the
+<a href="https://www.iana.org/assignments/media-types/media-types.xhtml">
+IANA Media Types</a> registry.
+                  </td>
+                </tr>
+                <tr>
+                  <td>`digestSRI`</td>
+                  <td>
+A cryptographic digest, as defined in [[[SRI]]].
+                  </td>
+                </tr>
+                <tr>
+                  <td>`digestMultibase`</td>
+                  <td>
+A cryptographic digest, as defined in [[[VC-DATA-INTEGRITY]]].
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+Each object associated with `relatedResource` MUST contain at least a
+`digestSRI` or `digestMultibase` value.
+          </dd>
+        </dl>
+
```

Comment:
This new section "smells" like defining a new type, in RDF sense, for a related resource. _At the moment_, in the vocabulary, the range of a `relatedResource` property is IRI, i.e., the only restriction on the property is that it is a an object property (as opposed to a datatype property). This bullet list seems to indicate that we want a new type (`RelatedResource`), in the domain a number of properties.

Do we want to formalize that in the vocabulary? Note that the last statement ("MUST contain at least a `digestSRI` or `digestMultibase` value") would require some OWL wizardry, which we probably should not include in our (simplified) vocabulary. Note also that this would require some changes in the DI vocabulary, because that is where `digestMultibase` is defined.

(If we agree these changes should happen, I am happy to come up with new PR-s at some point.)

Reply:
If we formalize that, we should probably formalize both `relatedResource` and its type in the Data Integrity vocabulary. It makes more sense for it to go over there than for it to be defined by the VCDM. If we make this decision, we'll need to make it quickly and affect the change quickly so that we can lock the tests down for CR.

Reply:
I am happy to do either way... in any case, both vocabularies will have to be updated.

I would propose to (1) have this PR merged and (2) have DI PR-s, especially https://github.com/w3c/vc-data-integrity/pull/258 handled and possibly merged before doing this. I do not want stupid merge conflicts...

Reply:
It seems that this addition is not editorial.  As one example, it pulls the use of digestMultibase into the VCDM specification, which I object to, per previous discussions.  Please remove these non-editorial additions.

Reply:
It was requested that this section was cleaned up.

In doing so, I consolidated all the normative statements to flow like the normative statements in the other sections. The `digestMultibase` value was an at-risk feature, along with `digestSRI` as specified here (see the at risk text starting with "Unification of cryptographic hash expression formats are under discussion" which was added when this section was added to the specification):

https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1464.html#integrity-of-related-resources

Please concretely identify which normative assertions didn't exist in the specification before the change.